scipy/signal/windows/barthann_doc

```diff
@@ -0,0 +1,20 @@
+import numpy as np
+from scipy import signal
+from scipy.fft import fft, fftshift
+import matplotlib.pyplot as plt
+
+window = signal.windows.barthann(51)
+plt.plot(window)
+plt.title("Bartlett-Hann window")
+plt.ylabel("Amplitude")
+plt.xlabel("Sample")
+
+plt.figure()
+A = fft(window, 2048) / (len(window)/2.0)
+freq = np.linspace(-0.5, 0.5, len(A))
+response = 20 * np.log10(np.abs(fftshift(A / abs(A).max())))
+plt.plot(freq, response)
+plt.axis([-0.5, 0.5, -120, 0])
+plt.title("Frequency response of the Bartlett-Hann window")
+plt.ylabel("Normalized magnitude [dB]")
+plt.xlabel("Normalized frequency [cycles per sample]")
```

Comment:
It looks like you've tried to rewrite the entire docstring example in a new file that wouldn't be used anyway instead of making changes to the original docstring code in `scipy/signal/windows/_windows.py`, and you're also missing the `>>>`  used in docstring examples.

What might be more useful is to chime on the issue we have open on this matter: gh-13049. There is some back and forth there about explicit and verbose vs. implicit imports, or even a "copy button" to inject the imports into the clipboard..